brave/src/main/java/brave/http/HttpParser.java

```diff
@@ -1,6 +1,6 @@
 package brave.http;
 
-import brave.Span;
+import brave.Tagger;
```

Comment:
notice.. now the http instrumentation customizers no longer reference span at all.